lms/djangoapps/courseware/grades.py

```diff
@@ -577,7 +598,7 @@ def _progress_summary(student, request, course, field_data_cache=None, scores_cl
 
     max_scores_cache.push_to_remote()
 
-    return chapters
+    return dict(chapters=chapters, scores=locations_to_scores)
```

Comment:
Sure. Is the convention to define such a class in the models.py file, or here near to where it's used?